src/main/scala-2/chisel3/simulator/LayerControl.scala

```diff
@@ -46,6 +80,20 @@ object LayerControl {
       }
     }
     override protected def shouldEnable(filename: String) = _shouldEnable(filename)
+
+    override protected def getLayerSubset(module: ElaboratedModule[_]): Seq[Layer] = {
+      val definedLayers = module.layers
+      layers.foreach {
+        case layer =>
```

Comment:
This is a minor nit but I think either of the following would be better, the `case layer =>` adds unnecessary tabbing:
```suggestion
      layers.foreach { layer =>
```
or
```suggestion
      for (layer <- layers) {
```

Reply:
Good call. Unnecessary tabbing should always be avoided. I went with option (1) in dd73577dc (as you likely expected I would do...).